src/theme/themeProperties.ts

```diff
@@ -11,6 +11,10 @@ const themeProperties = {
     fontXLarge: '26px',
     fontXXLarge: '36px',
     fontXXXLarge: '48px',
+    fontRemLarge: '2.25rem',
+    fontRemMedium: '1.5rem',
+    fontRemSmall: '1.125rem',
+    fontRemXSmall: '0.875rem',
```

Comment:
Could you add a comment here as a reminder to update this later and use only one system (either px or rem)? Perhaps we should use rem everywhere as that is a relative unit.